rewrite-java/src/main/java/org/openrewrite/java/tree/J.java

```diff
@@ -5275,4 +5279,373 @@ public Wildcard withBound(@Nullable JLeftPadded<Bound> bound) {
             }
         }
     }
+
+    @FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+    @EqualsAndHashCode(callSuper = false, onlyExplicitlyIncluded = true)
+    @RequiredArgsConstructor
```

Comment:
`@Data` is a shortcut for `@ToString`, `@EqualsAndHashCode`, `@Getter` on all fields, `@Setter` on all non-final fields, and `@RequiredArgsConstructor`